webui/react/src/components/Section.tsx

```diff
@@ -40,7 +40,7 @@ const Section: React.FC<Props> = (props: PropsWithChildren<Props>) => {
     <section className={classes.join(' ')} id={id}>
       {(props.title || props.options) && (
         <div className={css.header}>
-          {props.title && <h5 className={css.title}>{props.title}</h5>}
+          {props.title && !props.hideTitle && <h5 className={css.title}>{props.title}</h5>}
```

Comment:
**suggestion:** render this part only if `props.title` is defined? then we can get rid of `hideTitle` completely. I think `title` used to be required because it was used to generate default id but that is no longer the case, especially now you can pass in an id if you need control over that.

Reply:
I remember that and I thought I checked for that usage. let me take another look

Reply:
const defaultId = isString(props.title) ? toHtmlId(props.title) : generateAlphaNumeric();
  const id = props.id || defaultId;
  
  we do seem to still use it